scripts/build/cc/gcc.sh

```diff
@@ -396,6 +396,12 @@ do_gcc_core_backend() {
         extra_config+=("--disable-__cxa_atexit")
     fi
 
+    if [ "${CT_CC_GCC_TM_CLONE_REGISTRY}" = "y" ]; then
+        extra_config+=("--enable-tm-clone-registry")
+    else
+        extra_config+=("--disable-tm-clone-registry")
+    fi
```

Comment:
I'm wondering if we should just omit the else case.  I presume tm-clone-registry defaults to disabled in gcc.

Reply:
I'd presumed wrong. So we do need to disable this. Older GCCs will just see an option they don't recognize (that will emit a warning but should be otherwise harmless).